src/utils/helper-methods.ts

```diff
@@ -320,9 +320,9 @@ export const addTokenChainSuffix = (name: string, chain: string) => {
   return `${name.toLowerCase()}_${chain.charAt(0)}`;
 };
 
-export const getCurrencyAbbreviation = (name: string, chain: string) => {
-  return IsERCToken(name.toLowerCase(), chain.toLowerCase())
-    ? addTokenChainSuffix(name, chain)
+export const getCurrencyAbbreviation = (name: string, chain?: string) => {
+  return IsERCToken(name, chain)
+    ? addTokenChainSuffix(name, chain!)
```

Comment:
if chain is optional this condition could break some case

Reply:
Which case would it break? In `IsERCToken` we check to make sure chain is defined before returning `true`, so that addTokenChainSuffix always has a defined chain passed in to it.